src/database/DataTypes/Options.js

```diff
@@ -4,7 +4,16 @@
  */
 
 import Realm from 'realm';
-
+/**
+ * An Options object is a simple explanation of why a change or
+ * value was used in a situation. Used for stock takes, to
+ * provide a reason for having a stock take batch quantity differ
+ * from the stock take batch snapshot quantity.
+ * @property  {string}  id
+ * @property  {string}  title     A generic name field
+ * @property  {string}  type      The type of option. See below for types
```

Comment:
See below for types?

Reply:
fixed 5eff88b97ac298594602cfeeb649dc403f2c8406